bluemira/equilibria/coils/_grouping.py

```diff
@@ -514,8 +514,10 @@ def _get_coiltype(self, ctype):
         return coils
 
     def get_coiltype(self, ctype: Union[str, CoilType]):
-        """Get coil by coil type"""
-        return CoilGroup(*self._get_coiltype(ctype))
+        """Get coils matching coil type"""
+        if coiltype := self._get_coiltype(ctype):  # if matched coils list is not empty
+            return CoilGroup(*coiltype)
+        return None  # Don't return anything.
```

Comment:
The comments are not needed here
```suggestion
        if coiltype := self._get_coiltype(ctype):
            return CoilGroup(*coiltype)
        return None
```